test/system/cypress/e2e/regression_tests/backoffice/providers/providers.spec.ts

```diff
@@ -113,25 +113,25 @@ describe("Providers", () => {
     cy.contains("div.alert-success", message.successDeletionMessage).should("be.visible");
   });
 
-  it("shouldn't delete provider with services with draft status", () => {
+  it.skip("shouldn't delete provider with services with draft status", () => {
```

Comment:
Can you add a skip reason here? "According to new business logic, that requirement is no longer valid" 
Or just delete those tests